chainerx_cc/CMakeLists.txt

```
@@ -111,7 +111,7 @@ endif()
 
 # Compiler-specific workarounds
 
-if(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
+if(CMAKE_CXX_COMPILER_ID MATCHES "^(Clang|AppleClang)$")
```

Is the following not sufficient?
```suggestion
if(CMAKE_CXX_COMPILER_ID MATCHES "Clang|AppleClang")
```

---

I don't really have any preference here. I'll commit the suggestion if it's better for you.

---

That'd be good. For consistency with https://github.com/chainer/chainer/pull/5776/files/c6f2d9feae51ae5379e4dd0a9b8d116debd6bf6b#diff-1e5eafb3b2b05e7c616558d2154480a1R86

---

Sorry, please ignore this comment as `... MATCHES "Clang"` will match "AppleClang" too.